addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4120,9 +4120,12 @@ export class OdooEditor extends EventTarget {
                 restore(); // Make sure to make <br>s visible if needed.
             }
         }
+
+        const t_attrs = ['t-elif', 't-else', 't-esc', 't-foreach', 't-if', 't-out', 't-raw', 't-value']
```

Comment:
Thanks for the fix! Note that the convention when writing Javascript at Odoo is to use camel case and to always put the semicolon at the end of the line. 😉